README.md

```diff
@@ -41,77 +41,58 @@ Curious to see how it all works? Check out our [control flow diagrams](/docs/des
 
 Ready to quickly deploy the latest version of Azure Service Operator on your Kubernetes cluster and start exploring? Follow these steps.
 
-1. Make sure `kubectl` is configured to connect to the Kubernetes cluster you want to deploy Azure Service Operator to.
-    
-    To connect to an Azure Kubernetes Service cluster, you can use the below command:
 
-    ```console
-    az aks get-credentials -g <AKSClusterResourceGroup> -n <AKSClusterName>
-    ```
-
-1. Install cert-manager on the cluster using the following commands.
+1. Install [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster using the following command.
 
     ```console
-    kubectl create namespace cert-manager
-    kubectl label namespace cert-manager cert-manager.io/disable-validation=true
-    kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
+     kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
```

Comment:
Suggest removing additional indent, since it looks strange in output:

![image](https://user-images.githubusercontent.com/12575/86410940-dee61080-bd0f-11ea-8b6b-2095a9437c00.png)


```suggestion
    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
```